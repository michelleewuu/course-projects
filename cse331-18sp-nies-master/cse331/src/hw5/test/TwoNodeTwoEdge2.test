# Tests adding two nodes, one self-pointing edge 
# and one edge between two nodes to graph.

# Create a graph
CreateGraph two_nodes_two_edges_graph2

# Add two nodes
AddNode two_nodes_two_edges_graph2 a
AddNode two_nodes_two_edges_graph2 b

# Add two edges (including one self-pointing edge)
AddEdge two_nodes_two_edges_graph2 a a AA
AddEdge two_nodes_two_edges_graph2 a b AB

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
ListNodes two_nodes_two_edges_graph2
ListChildren two_nodes_two_edges_graph2 a
ListChildren two_nodes_two_edges_graph2 b