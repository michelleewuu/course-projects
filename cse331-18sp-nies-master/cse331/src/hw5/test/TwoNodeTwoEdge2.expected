# Tests adding two nodes, one self-pointing edge 
# and one edge between two nodes to graph.

# Create a graph
created graph two_nodes_two_edges_graph2

# Add two nodes
added node a to two_nodes_two_edges_graph2
added node b to two_nodes_two_edges_graph2

# Add two edges (including one self-pointing edge)
added edge AA from a to a in two_nodes_two_edges_graph2
added edge AB from a to b in two_nodes_two_edges_graph2

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
two_nodes_two_edges_graph2 contains: a b
the children of a in two_nodes_two_edges_graph2 are: a(AA) b(AB)
the children of b in two_nodes_two_edges_graph2 are: